src/CommonIncludes.h

```diff
@@ -29,9 +29,9 @@ FLARE.  If not, see http://www.gnu.org/licenses/
 #include <string>
 #include <vector>
 
-#include <SDL.h>
-#include <SDL_image.h>
-#include <SDL_mixer.h>
+#include "SDL.h"
```

Comment:
It shouldn't be necessary to change this. And changing it might break builds in other environments.

I'm not familiar with XCode, but there is likely a way to get it to point to the cloned SDL 2 headers.